requirements/system/user-client/functional.tex

```diff
@@ -0,0 +1,50 @@
+\section{Functional Requirements}
+\begin{itemize}
+  \item[UCF10] The user client shall implement all relevant command line options for the specified in Command Line Interface.
```

Comment:
"Relevant" is imprecise.
This makes it sound like we could arbitrarily decide to make commands optional.

Also, the sentence is grammatically wrong.
How about this:

The user client shall implement a command line interface as specified in chapter \ref{cli}.

You would then also need to add \label{cli} to the cli chapter.

Reply:
Oh yeah I missed some words, I meant `relevant command line options for the user client ...`.
I will add a link to the chapter as you suggested.